### PR TITLE
[model] support vision language model llava.

### PR DESCRIPTION
**Pull request by liutongxuan**

### CODE REVIEW

File: scalellm/_C/vlm_handler.pyi [hunk 1]
```diff
@@ -0,0 +1,55 @@
+from enum import Enum
+from typing import Callable, List, Optional
+
+from scalellm._C.output import RequestOutput
+from scalellm._C.sampling_params import SamplingParams
+
+class Priority(Enum):
```
Comment (guocuimi) [inline scalellm/_C/vlm_handler.pyi:7]: redefine Priority.

File: src/engine/batch.cpp [hunk 2]
```diff
@@ -49,6 +49,8 @@ void Batch::add(Sequence* sequence, uint32_t token_budget) {
   sequences_.push_back(sequence);
   token_budgets_.push_back(token_budget);
   budget_used_.push_back(0);
+
+  input_embedding_ = sequence->get_input_embedding();
```
Comment (guocuimi) [inline src/engine/batch.cpp:53]: sounds one input embedding for each sequence, if so, it should be a vector of input embeddings for the batch.

**Comment by guocuimi** [comment]
LGTM. Thank you for your hard work in getting the first VLM up and running. I've left some comments; I believe we should refactor to reduce redundant code between the worker and engine after landing this.

File: scalellm/_C/vlm_handler.pyi (hunk 1 above)
Comment (liutongxuan) [reply scalellm/_C/vlm_handler.pyi]: fixed

File: src/engine/batch.cpp (hunk 2 above)
Comment (liutongxuan) [reply src/engine/batch.cpp:53]: yes, in this pr only support one sequence + one image, and will support multiple sequences later.

**Pull request merged**